plugins/BLE/ble.py

```diff
@@ -1,2 +1,79 @@
-# yo deadass bluez sucks and my pi dies installing any bluetooth library
-# someone work on this pelase
\ No newline at end of file
+# Source: https://github.com/RapierXbox/ESP32-Sour-Apple
+# Licensed under GPL 3.0, we should be able to use this right? 
+# ESP32(Python) Sour Apple by @RapierXbox and @Amachik
+
+from core.plugin import BasePwnhyvePlugin
+from core.SH1106.screen import screenConsole, waitForKey
+import random
+import bluetooth._bluetooth as bluez
+from time import sleep
+import struct
+import socket
+import array
+import fcntl
+from errno import EALREADY
+
+screen = screenConsole(draw, disp, image)
+
+class PWNble(BasePwnhyvePlugin):
+    def loading(self):
+        for _ in range(3):
+            print("Starting iPhone BLE", end="", flush=True)
+            sleep(0.5)
+            print(".", end="", flush=True)
+            sleep(0.5)
+            print(".", end="", flush=True)
+            sleep(0.5)
+            print(".", end="", flush=True)
+            sleep(0.5)
+            print("\b\b\b   \b\b\b", end="", flush=True)
+
+    def iphoneBLE(self, GPIO):
+        try:
+            screen.start()
+            self.loading()
```

Comment:
not required, plus as it's printing to the log the user doesn't see this

```suggestion
            #self.loading()
```